schemas/app-sre/app-1.yml

```diff
@@ -27,9 +27,20 @@ properties:
     - OnBoarded        # The app is fully on-boarded, App SRE are carrying the pager
     - BestEffort       # The app is supported on a best effort basis
 
-  grafanaUrl:
-    type: string
-    format: uri
+  grafanaUrls:
+    type: array
+    description: |
+      List of links to Grafana Dashboard/Folders associated with the service. Please provide 
+      a title together with the link.
+    items:
+      type: object
+      properties:
+        title:
+          type: string
+        url:
+          type: string
+          format: uri
+      minItems: 1
```

Comment:
In order to make this object a bit more tight schema wise, I would add
```yaml
additionalProperties: false
required:

Reply:
:+1: